admin/src/main/resources/io/buoyant/admin/js/main-linkerd.js

```diff
@@ -51,6 +53,10 @@ require([
     adminPage.initialize().done(function() {
       new loggingConfig();
     });
+  } else if (window.location.pathname.endsWith("/usage")) {
```

Comment:
via meat bag conversation: it could be interesting to create a `main-usage.js` file that would be loaded in place of `main-linkerd.js`. Doesn't really solve the "where to serve static assets" problem, but would make this routing less of an eyesore